statistics_calculations.py

```diff
@@ -117,3 +117,23 @@ def plot_percentage(counter):
     ax.legend(loc='center left', bbox_to_anchor=(1, 0.5))
     plt.xlim(0,100)
     plt.show()
+
+def plot_word(df, word, regex=True):
+    """ Counts the use of a specific word by month, and plot by user
+        Be careful, word might actually be a regex
+    """
+    by_month = df[df["text"].str.contains(r"{}($|\s)".format(word))]
+
+    if not len(by_month):

```

Comment:
I think pandas [empty()](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.empty.html) is a better fit here

Reply:
Didn't know this function. That nice!